data/mods/Aftershock/items/items.json

```
@@ -17,7 +17,8 @@
   {
     "id": "afs_basin",
     "type": "GENERIC",
-    "name": { "ctxt": "container", "str": "basin" },
+    "name": { "str": "basin" },
+    "category": "container",
```

```suggestion
    "name": { "ctxt": "container", "str": "basin" },
```
"ctxt" means the string is translated with a context to distinguish it from other strings with the same text but different meaning.